postgres-driver/application.go

```diff
@@ -60,8 +60,12 @@ const (
 	VALUES (:application_id, :daily_limit)`
 	updateApplication = `
 	UPDATE applications
-	SET name = COALESCE($1, name), user_id = COALESCE($2, user_id), status = COALESCE($3, status), updated_at = $4
```

Comment:
I don't think you should delete status from update, maybe some statuses are not direcly related with remove

Reply:
Yeah you're right. Not currently used for anything else but probably will be.